macOS/UnitTests/Onboarding/ContextualOnboarding/ContextualOnboardingStateMachineTests.swift

```diff
@@ -646,6 +726,34 @@ class ContextualOnboardingStateMachineTests: XCTestCase {
         XCTAssertEqual(stateMachine.state, .showMajorOrNoTracker)
     }
 
+    func test_OnUpdateStateFor_VisitsSite_WhenOnFireUsedTryASearchShownh_AndNoTrackers_ThenStateIsShowMajorOrNoTracker() {
```

Comment:
typo: 
```suggestion
    func test_OnUpdateStateFor_VisitsSite_WhenOnFireUsedTryASearchShown_AndNoTrackers_ThenStateIsShowMajorOrNoTracker() {
```
Should also this be `_AndMajorTracker`?